services/web-app/pages/designing/figma.mdx

```diff
@@ -51,9 +57,9 @@ Open the menu and select **Libraries** from the list (B).
 </Column>
 </Row>
 
-Then in the Libraries modal that appears, find the team called `[NEW] IBM Design Systems` (C) and
-switch the toggle beside a themed library, for example the `White theme - Carbon Design System` to
-on (D).
+In the Libraries modal that appears, find the team called `IBM Design Systems` (C) and switch the
+toggle beside a themed library. For example, switch the `(v11) White Theme - Carbon Design System`
+to 'on' (D).
```

Comment:
Did you want on to be in quotes or in a snippet? If a snippet then do this:

```suggestion
to `on` (D).
```

Reply:
I had meant it to be in quotes like you would typically do in writing instructions but I like the snippet better!